tests/ui/consts/const-mut-refs/issue-76510.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0764]: mutable references are not allowed in the final value of constants
+error[E0764]: mutable borrows of lifetime-extended temporaries are not allowed in constants
```

Comment:
This is still somewhat abbreviated... lifetime extension "within" constants is fine, it's just lifetime extension at the top-level scope where we restrict borrows. I couldn't think of a good way to concisely state this.

Reply:
https://doc.rust-lang.org/nightly/reference/destructors.html#r-destructors.scope.lifetime-extension.static also just says "static and const items" without saying anything about "top-level scope".

Reply:
> "mutable borrows of lifetime-extended temporaries in the top-level scope"?

I've gone for something like this now.